recipes/carna/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+  name: carna
+  version: "1.3.1"
+
+about:
+  home: http://www.bioinf.uni-freiburg.de/Software/Carna/
+  license: GPL
+  license_file: COPYING
+  summary: Multiple alignment of RNAs
+
+build:
+  number: 0
+  skip: True # [osx]
```

Comment:
osx cannot work yet, since currently the dependencies cannot be build under osx. In particular, I need ViennaRNA, then locarna, then carna. I'll try gecode